docs/api.md

```diff
@@ -490,3 +490,62 @@ empty
 | ----- | -------- | 
 | 200   | |
 | 500 | something went wrong (see logs of the simulator server) |
+
+## Export
+
+Export current scheduler resources in `json`.
+
+### HTTP Request
+
+`GET /api/v1/export`
+
+
+### Response
+
+`Content-Type: application/json; charset=UTF-8`
+
+```
+type ResourcesApplyConfiguration struct {
```

Comment:
This should be written in the json format, not the Go format.

Reply:
In that case, the other descriptions also should be fixed to JSON, shouldn't they?
https://github.com/kubernetes-sigs/kube-scheduler-simulator/blob/4abe7f1946fa69a7abb2243ca08750b4b7eeff76/docs/api.md?plain=1#L28

Reply:
@sanposhiho 
There are a lot of parameters in the upload/download JSON data. 
And this document will be hard to read.

How about saving a sample of the upload/download data in a separate file? (and paste the link in here.)

Reply:
> In that case, the other descriptions also should be fixed to JSON, shouldn't they?

It's a fair point. So, okay, for now, let's write it in Go instead of JSON for now. We can improve this doc later if we want.

However, as it is, we don't know which package `v1` or `confstoragev1` represents. So, instead of writing Go struct directly here, I think it would be better to post a link to `ResourcesApplyConfiguration`. (server/handler/export.go:L20)


> How about saving a sample of the upload/download data in a separate file?

Yes. It will be another help for users.